co_simulation/validation/dam_break_flex_wall/source/ProjectParametersCoSim.json

```diff
@@ -0,0 +1,154 @@
+{
+    "problem_data" :
+    {
+        "start_time" : 0.0,
+        "end_time" : 1.0,
+        "echo_level" : 0,
+        "print_colors" : true,
+        "parallel_type" : "OpenMP"
+    },
+    "solver_settings" :
+    {
+        "type" : "coupled_solvers.gauss_seidel_strong",
+        "echo_level" : 0,
+        "num_coupling_iterations" : 20,
+        "convergence_accelerators" : [
+            {
+                "type"      : "aitken",
+                "solver"    : "pfem",
+                "data_name" : "vel"
+            },
+            {
+                "type"      : "aitken",
+                "solver"    : "pfem",
+                "data_name" : "disp"
```

Comment:
I don't remember, did you also try with different accelerators?

Reply:
I did try with the IQN-ILS accelerator. By only changing lines 22 to 24, it was possible without other modifications to the code. It reduced the average iteration number from 12 to 8, saving around 6 hrs of sim time (total time with aitken relaxation in my remote pc is around 1 day 13 hrs), but... the displacement field was a bit oscillatory. This is actually something I would have liked to investigate more... but there was no more time left for my thesis.